src/java/org/apache/cassandra/index/SecondaryIndexManager.java

```diff
@@ -1175,7 +1175,7 @@ public Index.QueryPlan getBestIndexQueryPlanFor(RowFilter rowFilter)
         return selected;
     }
 
-    private static String commaSeparated(Collection<Index> indexes)
+    private static String commaSeparated(Collection<? extends Index> indexes)
```

Comment:
Why do we need this change? By the way, I really like how such a big patch for indexes doesn't need changes on the `SecondaryIndexManager` :)

Reply:
Sorry, that was left over from an attempt to make some of the interfaces more generic.